nsds_lab_to_nwb/metadata/resources/list_of_stimuli.yaml

```diff
@@ -20,14 +20,14 @@ tone150:
     parameter_path: 'Tone150/freq_resp_area_stimulus_signal_flo500Hz_fhi32000Hz_nfreq30_natten1_nreps150_fs96000.mat'
 
 timit:
-    alt_names: []
+    alt_names: ['Timit998']
     description: ''
-    audio_path: 'TIMIT/timit998s.wav'
-    marker_path: 'TIMIT/timit998m.wav.wav'
+    audio_path: 'TIMIT/timit998/timit998s.wav'
+    marker_path: 'TIMIT/timit998/timit998m.wav'
```

Comment:
Should we just move the .wav files up a level to be more similar to other stimuli? Are there other timit besides 998?

Reply:
There are in fact multiple timit stimuli files, but I don't think we use them for current experiments. One possibility is to flatten the directory structure so that Tone, timitA, timitB, etc.. are all level on the same level as opposed to what was already done (Tone, TIMIT/timitA, TIMIT/timitB).